frontend/packages/ceph-storage-plugin/src/components/bucket-class/review-utils.tsx

```diff
@@ -12,7 +12,7 @@ export const ReviewListTitle: React.FC<ReviewListTitleProps> = ({ text }) => (
 
 type ReviewListTitleProps = { text: string };
 
-export const ReviewListBody: React.FC<ReviewListBodyProps> = ({ children }) => <dd>{children}</dd>;
+export const ReviewListBody = ({ children }) => <dd>{children}</dd>;
```

Comment:
I don't see point of reusing that component. We don't have to do  to show success or failed icon. There is only one validation for bucket class name and rest of specific validations are done on page. Reusing this would require adding few more conditionals to remove the unnecessary parts, changing it in review pages and  test for all bucket classes (Standard and Namespace) . It seems like whole lot of trouble,  I'll like to keep it simple as it is now. 

![Screenshot from 2021-04-06 18-16-05](https://user-images.githubusercontent.com/57935785/113712962-667c0800-9704-11eb-8cae-27feb882a74c.png)

Reply:
Those are optional parameters. you can just pass `children` similar to current.

Reply:
ok, i have removed the title and body. Needed to pass hideIcon to all body components

Reply:
I dont think you need to pass that, its optional.

Reply:
The success icon is showing up without that prop...